docs/development/reuse-api.md

```diff
@@ -0,0 +1,36 @@
+# REUSE API compliance
+
+We are using the [REUSE API](https://api.reuse.software/) to check and display the compliance with the REUSE guidelines.
+
+To run the REUSE linter to ensure the compliance of the project in case you added new files to the project you need to 
+runt the following steps:
+
+1. Install the `reuse` tool
+
+```shell
+# setup a virtual python environment in a location of your choice
+virtualenv -p python3 dev
+# active your python environment
+source dev/bin/activate
+# install the reuse tool cli
+pip install reuse
+```
+
+2. Run the `resue` linter
+
+```shell
+reuse lint
```

Comment:
Can you adapt `make check` to check if the project is reuse-compliant and remove the check for the license header?
https://github.com/gardener/gardener/blob/d395f33500d91c144202d15b9760e39a3783a973/Makefile#L167
I guess if the project is reuse-complaint only if all files have a license?

Reply:
Unfortunately the `reuse` tool is a python app and I am not a huge fan of instrumenting `python` style tools installation via `Makefile`. Maybe a Docker based solution might help here. Maybe this topic needs to be addressed in a seperate PR.

@MartinWeindel you enabled reuse for some of your projects. Do you have some instrumentation of the reuse tool in your `Makefile`s?